src/eventManagerSvc/settings.py

```diff
@@ -3,12 +3,12 @@
 
 class Settings(BaseSettings):
     server_host: str = '0.0.0.0'
-    server_port: int = DEPLOY_CONNECTION_PORT000
+    server_port: int = 8080
     database_url: str
     player_svc_host = '0.0.0.0'
-    player_svc_port = DEPLOY_CONNECTION_PORT001
+    player_svc_port = 8080
     event_svc_host = '0.0.0.0'
-    event_svc_port = DEPLOY_CONNECTION_PORT002
+    event_svc_port = 8080

```

Comment:
It use same port for all three services and that can't be right

Reply:
Yes, I just use same port for local debug. We need to rework our deployment scripts and get host/port from environment variables or something like this if we wouldn't use docker.